provider/github/utils.go

```diff
@@ -71,36 +64,25 @@ func castHash(sha1 *string) plumbing.Hash {
 	return plumbing.NewHash(*sha1)
 }
 
-func castPullRequestEvent(
-	r *lookout.RepositoryInfo,
-	e *github.Event, pr *github.PullRequestEvent,
-) *lookout.ReviewEvent {
-
-	if pr.PullRequest == nil && pr.PullRequest.GetID() != 0 {
-		log.Warningf("missing pull request information in pull request event")
-		return nil
-	}
-
+func castPullRequestEvent(r *lookout.RepositoryInfo, pr *github.PullRequest) *lookout.ReviewEvent {
```

Comment:
I'd rename this to `castPullRequest`, since now it takes a `PullRequest` instead of a `PullRequestEvent`